pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -360,6 +371,7 @@ in with passthru; stdenv.mkDerivation (finalAttrs: {
     "--enable-framework=${placeholder "out"}/Library/Frameworks"
   ] ++ optionals (pythonAtLeast "3.13") [
     (enableFeature enableGIL "gil")
+    (enableFeature enableJIT "experimental-jit")
```

Comment:
We could make `--enable-experimental-jit=yes-off` the default, as in fedora or conda-forge.
In this case, JIT would be enabled when the user sets `PYTHON_JIT=1` without rebuild.
What do you think?
fedora: https://src.fedoraproject.org/rpms/python3.13/blob/rawhide/f/python3.13.spec#_148
conda-forge: https://github.com/conda-forge/python-feedstock/blob/12d71f4a8841a10281c986e82fc4f7c3cab10329/recipe/build_base.sh#L270-L274

But I don't know if it's reproducible, and we need to build it successfully first.🤷

Reply:
Yes, sounds like a reasonable choice for an experimental feature.